drag_and_drop_v2/public/js/drag_and_drop.js

```diff
@@ -48,7 +65,7 @@ function DragAndDropTemplates(configuration) {
     };
 
     var itemContentTemplate = function(item) {
-        var item_content_html = item.displayName;
+        var item_content_html = gettext(item.displayName);
```

Comment:
That's a good question. My understanding is that it's being set during runtime as this will depend on the XBlock settings in Studio. I think we need to refer to the previous PR and ticket as the developer who took this work left.

Reply:
We provide translations of the [default data](https://github.com/edx-solutions/xblock-drag-and-drop-v2/blob/f0aa4e1e4a96c0ac93a83177c007456735c64a2a/drag_and_drop_v2/default_data.py) for the demo DnDv2 XBlock, and so that's why this text is being translated.  This makes the demo DnDv2 XBlock useful for non-English speakers when they need to understand how the XBlock works.

Once someone edits the DnDv2 XBlock and adds their own content, then it can't be automatically translated anymore, and so that `gettext` call wouldn't change the displayed content.

Though if the text the user chooses happens to be in the list of translated strings, then it could be translated, and this may not be the desired behaviour.  But the benefits of having a translated demo I think outweigh this risk.

Does this make sense?  Do you agree/disagree?